pkg/appfile/storage/driver/driver.go

```diff
@@ -0,0 +1,26 @@
+package driver
+
+import (
+	"github.com/oam-dev/kubevela/pkg/appfile"
+	"github.com/oam-dev/kubevela/pkg/appfile/template"
+)
+
+// Driver is mutli implement interface
+type Driver interface {
+	// List applications
+	List(envName string) ([]*RespApplication, error)
+	// Save application
+	Save(app *RespApplication, envName string) error
```

Comment:
Does `Save` means creating an application?

Reply:
No, just save to app dir.
https://github.com/oam-dev/kubevela/blob/master/pkg/commands/up.go#L206